bin/dispatch-conf

```diff
@@ -155,11 +155,25 @@ class dispatch:
             if not os.path.isdir(os.path.dirname(path)):
                 continue
 
+			# check whether find command is the GNU version
+			# if not, use BSD-style find arguments
+            s = subprocess.run(["find", "--version"], capture_output=True)
```

Comment:
A presence of `--version` option is hardly sufficient to guarantee that it's GNU find.